.github/workflows/cli-build.yml

```diff
@@ -1,4 +1,4 @@
-name: ci
+name: CLI Build
```

Comment:
this actually compiles the entire repo, including the typescript generators and we have PR requirements set on `ci/compile` `ci/lint` etc

Reply:
reverting this change. if we can be more descriptive, that'd be great!